recipes-rte/rte-tests/files/cukinia.conf

```diff
@@ -0,0 +1,47 @@
+export CUKINIA_ROOT=/etc/cukinia
+
+cukinia_kconf GPIO_SYSFS y
+cukinia_kconf SPI_SPIDEV y
+cukinia_kconf PINCTRL_MCP23S08 m
+
+# GPIO expander
+cukinia_test -d /sys/class/gpio/gpiochip400
+
+cukinia_test -d /sys/class/gpio/gpio199
+cukinia_test -d /sys/class/gpio/gpio11
+cukinia_test -d /sys/class/gpio/gpio12
+cukinia_test -d /sys/class/gpio/gpio199
+cukinia_test -d /sys/class/gpio/gpio400
```

Comment:
@PLangowski These must be updated

Reply:
bc6884190301773097e97b3ddb40e133d1f39f30